src/prometheus-stack/chart/templates/coredns-netpol.yaml

```
@@ -0,0 +1,26 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+{{- if (eq .Values.addRke2CorednsNetpol.enabled true) }}
```

nit: with helm we can just directly use the conditional unless I'm missing something special here?

```suggestion
{{- if .Values.addRke2CorednsNetpol.enabled }}
```

If you do make this change I might advocate for `rke2CorednsNetpol.enabled` (dropping the `add` as well, since it feels redundant with enabled).

---

I like this suggestion, I'll push up that change